src/client/pages/farm.vue

```diff
@@ -47,14 +62,19 @@ export default {
 
 .farm-page__panel {
   position: relative;
-  min-width: 50%;
+  width: 0;
   height: 100%;
   overflow-y: auto;
   background-color: var(--background-light);
+  transition: .5s width cubic-bezier(0.075, 0.82, 0.165, 1);
```

Comment:
Never animate width. Only use animate/transition on elements that only [trigger layout](https://csstriggers.com/) (not paint or composite).

You are better of animating in the panel from the left:
```css
.farm-page__panel {
  transform: translate(0);
  transition: .5s transform cubic-bezier(0.075, 0.82, 0.165, 1);
}
.farm-page__panel--loading {
  transform: translate(-100%);
}